app/code/core/Mage/Adminhtml/Model/System/Config/Source/Admin/Page.php

```diff
@@ -124,7 +124,7 @@ protected function _buildMenuArray(Varien_Simplexml_Element $parent=null, $path=
 
         uasort($parentArr, array($this, '_sortMenu'));
 
-        while (list($key, $value) = each($parentArr)) {
+        foreach ($parentArr as $key => $value) {
```

Comment:
Since PHP7.3, instead of loop and 6 lines of code, it can be simplified with:
```php
if ($parentArr) {
    $parentArr[array_key_last($parentArr)]['last'] = true;
}

Reply:
Our composer file says, we still support 7.0.
And for migration reasons we should stay with compatibility for 7.2.

Reply:
Yeah, PHP7.2 support is important